configure incorrectly errors if the python json module is unavailable, but simplejson is availabe

As far as I can tell, if you attempt to configure with an older version of python (with simplejson, but not json, available) then the configuration script fails as follows:

```
checking python module: json... no
configure: error: failed to find required module json
```

This is because configure.ac incorrectly claims that the lack of the json module is a fatal error, so the check below for simplejson never runs.

--- a/configure.ac
+++ b/configure.ac
@@ -107,7 +107,7 @@ AC_ARG_ENABLE([64-bit],
 
 AM_PATH_PYTHON([2.5])
 
-AX_PYTHON_MODULE([json], [Need JSON])
+AX_PYTHON_MODULE([json], [])
 AS_IF([test "x$HAVE_PYMOD_JSON" = "xno"],
       [AX_PYTHON_MODULE([simplejson])])
 AS_IF([test "x$HAVE_PYMOD_JSON" = "xno" && test "x$HAVE_PYMOD_SIMPLEJSON" = "xno"],

